tests/integration_tests/test_pytorch_lightning.py

```diff
@@ -87,3 +89,13 @@ def objective(trial: optuna.trial.Trial) -> float:
     study.optimize(objective, n_trials=1)
     assert study.trials[0].state == optuna.trial.TrialState.COMPLETE
     assert study.trials[0].value == 1.0
+
+
+def test_pytorch_lightning_pruning_callback_monitor_is_invalid() -> None:
+
+    study = optuna.create_study(pruner=DeterministicPruner(True))
+    trial = create_running_trial(study, 1.0)
+    callback = PyTorchLightningPruningCallback(trial, "InvalidMonitor")
+
+    with pytest.warns(UserWarning):
+        callback.on_validation_end(0, {"loss": 1.0})
```

Comment:
```suggestion

    trainer = pl.Trainer(
        min_epochs=0,  # Required to fire the callback after the first epoch.
        max_epochs=1,
        checkpoint_callback=False,
        callbacks=[callback],
    )
    model = Model()

    with pytest.warns(UserWarning):
        callback.on_validation_end(trainer, model)

```

Reply:
I think the current `on_validation_end`'s arguments look incorrect. How about this? Locally, `tests/integration_tests/test_pytorch_lightning.py` is passed.